cmd/minikube/cmd/start.go

```diff
@@ -161,6 +161,8 @@ func runStart(cmd *cobra.Command, _ []string) {
 	}
 	defer pkgtrace.Cleanup()
 
+	viper.SetDefault(config.DefaultClusterDNSDomain, constants.DefaultClusterDNSDomain)
```

Comment:
this should be func setupViper()

Reply:
I was trying to set the default only in the subcommand (`start`) that uses it. The `setupViper` function applies to all the subcommands. I am happy to move it there. Is this your recommendation?